app/uk/gov/hmrc/apinotificationpull/connectors/EnhancedApiNotificationQueueConnector.scala

```diff
@@ -31,6 +30,10 @@ class EnhancedApiNotificationQueueConnector @Inject()(config: ServiceConfigurati
 
   private lazy val serviceBaseUrl: String = config.baseUrl("api-notification-queue")
 
+  def getAllNotificationsBy(notificationStatus: NotificationStatus.Value)(implicit hc: HeaderCarrier): Future[Notifications] = {
+    http.GET[Notifications](s"$serviceBaseUrl/notifications/${notificationStatus.toString}")
+  }
+
```

Comment:
I think we should return a type of `Future[Either[ErrorResponse, Notification]]`